### PR TITLE
BaSP-TG-131: Delete account employee

### PR DESCRIPTION
**Pull request by matescarabino**
- Add functionality to delete form button
- Show confirmation and message buttons
- Once deleted, redirect to login

How to test?
- login as an Employee
- click on the delete account button
- test the correct functionality
- watch the result on mongoDB (isDeleted field should be true)
- after testing is done revert the changes on mongoDB

### CODE REVIEW

File: src/Components/Employees/Profile/index.js [hunk 1]
```diff
@@ -61,13 +62,19 @@ const EmployeesProfile = () => {
...
+        : dispatch(logout()),
+        dispatch(confirmModalClose());
+    } else {
+      dispatch(deleteEmployee(id, token));
       dispatch(confirmModalClose());
+    }
   };
 
   const onCancel = () => {
+    isDelete === true ? setIsDelete(false) : null;
```
Comment (CristianBucci) [inline src/Components/Employees/Profile/index.js:77]: Here you can use `isDelete && setIsDelete(false)`

File: src/Components/Employees/Profile/index.js [hunk 2]
```diff
@@ -86,13 +93,20 @@ const EmployeesProfile = () => {
 
   const modalFunction = () => {
     modalContent.title.includes('SUCCESS');
+    isDelete === true ? dispatch(logout()) : null;
```
Comment (CristianBucci) [inline src/Components/Employees/Profile/index.js:96]: Same as above!

**Review by CristianBucci — APPROVED** [review]
Everything looks right!

File: src/Components/Employees/Profile/index.js [hunk 3]
```diff
@@ -86,13 +93,20 @@ const EmployeesProfile = () => {
...
     dispatch(messageModalClose());
   };
 
   const resetForm = () => {
     reset(formValues);
   };
 
+  const deleteAccount = () => {
+    setIsDelete(true);
+    const content = `Are you sure you want to delete your Account?`;
```
Comment (Ansitox) [inline src/Components/Employees/Profile/index.js:106]: Change backticks

File: src/redux/employees/reducer.js [hunk 4]
```diff
@@ -78,7 +78,9 @@ const employeesReducer = (state = INITIAL_STATE, action) => {
       return {
         ...state,
         isLoading: false,
-        list: [...state.list.filter((employee) => employee._id !== action.payload)]
+        list: [...state.list.filter((employee) => employee._id !== action.payload)],
+        modalContent: { title: 'SUCCESS!', content: `Account Successfully DELETED` },
```
Comment (Ansitox) [inline src/redux/employees/reducer.js:82]: Change backticks

**Review by Ansitox — APPROVED** [review]

File: src/Components/Employees/Profile/index.js (hunk 3 above)
Comment (matescarabino) [reply src/Components/Employees/Profile/index.js:106]: Done

File: src/redux/employees/reducer.js (hunk 4 above)
Comment (matescarabino) [reply src/redux/employees/reducer.js:82]: Done

**Pull request merged**